src/app/modules/card-cluster/card-cluster.component.ts

```diff
@@ -1,133 +1,132 @@
 import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
 import { CardClusterModel } from './card-cluster.models';
 
-    @Component({
-        selector: 'app-card-cluster',
-        templateUrl: './card-cluster.component.html',
-        styleUrls: ['./card-cluster.component.css']
-    })
+@Component({
+  selector: 'app-card-cluster',
+  templateUrl: './card-cluster.component.html',
+  styleUrls: ['./card-cluster.component.css'],
+})
 export class AppCardClusterComponent implements OnInit {
+  @Input() cardArray: CardClusterModel;
 
-    @Input() cardArray: CardClusterModel;
+  // The following inputs have been deprecated and moved into the CardClusterModel object
+  @Input() buttonClicked = 'CardCluster0'; // DEPRECATED
+  @Input() dataAutoId: string; // DEPRECATED
+  @Input() resizeTimeout: number;
```

Comment:
most of the settings for the card cluster were moved into the CardClusterModel class as properties of that class.  I think it would be cleaner to move this variable into that class too.